From c71085bcea25ab5e46ee1ca93c30da6a0bb6bd72 Mon Sep 17 00:00:00 2001
From: Matthew Hrehirchuk <49077192+matthew-hre@users.noreply.github.com>
Date: Sun, 23 Feb 2025 16:35:59 -0700
Subject: [PATCH] update btusb.c

---
 drivers/bluetooth/btusb.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
index 90966dfbd2781f..ef07e54e6ac2df 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -519,6 +519,12 @@ static const struct usb_device_id quirks_table[] = {
 						     BTUSB_WIDEBAND_SPEECH },
 
 	/* Realtek 8852BE Bluetooth devices */
+
+        /* stupid budget asus mobo... stupid stupid... */
+	{ USB_DEVICE(0x0489, 0xe112), .driver_info = BTUSB_REALTEK |
+						     BTUSB_WIDEBAND_SPEECH },
+
+
 	{ USB_DEVICE(0x0cb8, 0xc559), .driver_info = BTUSB_REALTEK |
 						     BTUSB_WIDEBAND_SPEECH },
 	{ USB_DEVICE(0x0bda, 0x4853), .driver_info = BTUSB_REALTEK |

